packages/apps/graph/generated-schema.graphql

```diff
@@ -384,11 +384,17 @@ type Query {
   """
   fungibleAccount(accountName: String!, fungibleName: String): FungibleAccount
 
+  """Retrieve an account by public key."""
+  fungibleAccountsByPublicKey(publicKey: String!): [FungibleAccount!]
```

Comment:
Don't we want the fungiblename as a parameter here as well?

Reply:
The thing with that is that a fungible can have its own code to create accounts. I don't think its guaranteed that we can find created account using `${$fungibleName}.transfer-create`.

Can you confirm this @alber70g